shadowsocks/stream.go

```diff
@@ -37,6 +37,8 @@ type Writer interface {
 type shadowsocksWriter struct {
 	writer   io.Writer
 	ssCipher shadowaead.Cipher
+	// Wrapper for input that arrives as a slice.
+	input bytes.Reader
```

Comment:
Maybe rename to `byteWrapper` to be more explicit? `input` is ambiguous with the io.Reader.

Reply:
Done